gradlew.bat

```diff
@@ -1,90 +1,90 @@
-@if "%DEBUG%" == "" @echo off
```

Comment:
This was changes by gradle not me. If I don't commit it now then it will create noise for everyone else in the future.